packages/mui-base/src/Tabs/Root/TabsRoot.test.tsx

```diff
@@ -153,7 +153,7 @@ describe('<Tabs.Root />', () => {
             tabElements[index].id,
           );
 
-          await act(async () => {
+          await act(() => {
```

Comment:
Having an `async` here breaks 200+ other tests.

Reply:
Oh strange, there is some black magic 😅